noir/noir-repo/noir_stdlib/src/grumpkin_scalar_mul.nr

```diff
@@ -1,6 +0,0 @@
-use crate::grumpkin_scalar::GrumpkinScalar;
```

Comment:
Decided to nuke this as it was used only in 1 place in aztec-packages and it's completely unnecessary.